Dockerfile

```diff
@@ -4,11 +4,14 @@ RUN zypper -n update && \
     zypper -n install curl jq openssl nginx tar gzip sudo
 
 # Get Istio
-RUN curl -L https://istio.io/downloadIstio | ISTIO_VERSION=${ISTIO_VERSION} sh -
+ARG TARGETOS
```

Comment:
Can we please leave the logic of arch to istioctl? since we don't want to have that logic when istioctl already handles it.

Reply:
I had left it in to be less host-arch dependent, but in hindsight that only matters when cross-compiling. Since the build suite provides native arm64 builds, so my changes were redundant.